NAIP and Discover blog post

rkelson opened the pull request:
this PR needs to be reviewed and okay'd before being merged.
I will add a comment when the PR is okay'd and ready to merge.

The merge needs to happen early morning of 2/23/17 after Calvin makes some changes to Discover at 8 am eastern time

steveoh commented:
@stdavis will you please merge this 10 minutes before 8 tomorrow! Thanks.

steveoh commented on _posts/2017-02-21-new-services-on-discover.md:
```diff
@@ -0,0 +1,41 @@
+as this could take several minutes. Another option is to completely clear your ArcMap cache by going
+to `Customize -> ArcMap Options -> Display Cache -> Clear Cache`.  
+
+The service additions and changes are listed below. We also renamed most of the services. The renaming of the services should not have any effect on projects or applications that contained any of the services previous to the renaming. The renaming is on the layer's display title only a does not have any effect on the urls or request parameters for the service.
```
> layer's display title only a does not have any effect on the urls or ...

should say `only and does`

steveoh commented on _posts/2017-02-21-new-services-on-discover.md:
```diff
@@ -0,0 +1,41 @@
+ now utilizes scale dependent rendering to display the color balanced NAIP imagery down to cache level 16,
+ then renders the Google imagery at level 17 (starting at approx. 1:6,500) and down to level 20.
+
+ - **Google** (licensed)- This new service is the replacement for the old **utah** service and has the
```
Your above text has a space before the `-` and the others don't. would you update those.

steveoh commented on _posts/2017-02-21-new-services-on-discover.md:
```diff
@@ -0,0 +1,41 @@
+
+ - **Basemap-Hybrid** (licensed)- This new service was created for convenience and performance for a dynamic mash-up of vector features with a transparent background from the **Basemap-Overlay** and the **Utah** imagery service.
+
+More information about the services from Discover can be found [here]({{"/data/sgid-base-map-services-arcmap/" | prepend:site.baseurl}}).
```
`here` links are kind of rookie. Could you update that to something like 

More information about the services from Discover can be found in the [Using SGID Base Map & Imagery Services in ArcMap](#)

rkelson commented:
thanks for the second set of eyes

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

On Wed, Feb 22, 2017 at 5:04 PM, steveoh <notifications@github.com> wrote:

> *@steveoh* requested changes on this pull request.
> ------------------------------
>
> In _posts/2017-02-21-new-services-on-discover.md
> <https://github.com/agrc/gis.utah.gov/pull/384#discussion_r102606778>:
>
> > +- agrc
> +- discover
> +- google
> +- naip
> +- sgid
> +
> +---
> +
> +Recently AGRC made some changes to the services available from our cloud server Discover (discover.agrc.utah.gov).
> +To see these new enhancements you may need to _refresh_ the connection you use to access Discover. ArcMap users can
> +right-click your connection to Discover in ArcCatalog and select `Refresh`. You might also want to clear your local cache.
> +ArcMap users can go to the service’s `Layer Properties -> Cache` tab and select `Clear Local Cache Now`. Be patient
> +as this could take several minutes. Another option is to completely clear your ArcMap cache by going
> +to `Customize -> ArcMap Options -> Display Cache -> Clear Cache`.
> +
> +The service additions and changes are listed below. We also renamed most of the services. The renaming of the services should not have any effect on projects or applications that contained any of the services previous to the renaming. The renaming is on the layer's display title only a does not have any effect on the urls or request parameters for the service.
>
> layer's display title only a does not have any effect on the urls or ...
>
> should say only and does
> ------------------------------
>
> In _posts/2017-02-21-new-services-on-discover.md
> <https://github.com/agrc/gis.utah.gov/pull/384#discussion_r102607267>:
>
> > +---
> +
> +Recently AGRC made some changes to the services available from our cloud server Discover (discover.agrc.utah.gov).
> +To see these new enhancements you may need to _refresh_ the connection you use to access Discover. ArcMap users can
> +right-click your connection to Discover in ArcCatalog and select `Refresh`. You might also want to clear your local cache.
> +ArcMap users can go to the service’s `Layer Properties -> Cache` tab and select `Clear Local Cache Now`. Be patient
> +as this could take several minutes. Another option is to completely clear your ArcMap cache by going
> +to `Customize -> ArcMap Options -> Display Cache -> Clear Cache`.
> +
> +The service additions and changes are listed below. We also renamed most of the services. The renaming of the services should not have any effect on projects or applications that contained any of the services previous to the renaming. The renaming is on the layer's display title only a does not have any effect on the urls or request parameters for the service.
> +
> + - **Utah** (licensed) - This service is now a mix of the most recent NAIP 1-meter and Google 6-inch imagery. The service
> + now utilizes scale dependent rendering to display the color balanced NAIP imagery down to cache level 16,
> + then renders the Google imagery at level 17 (starting at approx. 1:6,500) and down to level 20.
> +
> + - **Google** (licensed)- This new service is the replacement for the old **utah** service and has the
>
> Your above text has a space before the - and the others don't. would you
> update those.
> ------------------------------
>
> In _posts/2017-02-21-new-services-on-discover.md
> <https://github.com/agrc/gis.utah.gov/pull/384#discussion_r102607535>:
>
> > +ArcMap users can go to the service’s `Layer Properties -> Cache` tab and select `Clear Local Cache Now`. Be patient
> +as this could take several minutes. Another option is to completely clear your ArcMap cache by going
> +to `Customize -> ArcMap Options -> Display Cache -> Clear Cache`.
> +
> +The service additions and changes are listed below. We also renamed most of the services. The renaming of the services should not have any effect on projects or applications that contained any of the services previous to the renaming. The renaming is on the layer's display title only a does not have any effect on the urls or request parameters for the service.
> +
> + - **Utah** (licensed) - This service is now a mix of the most recent NAIP 1-meter and Google 6-inch imagery. The service
> + now utilizes scale dependent rendering to display the color balanced NAIP imagery down to cache level 16,
> + then renders the Google imagery at level 17 (starting at approx. 1:6,500) and down to level 20.
> +
> + - **Google** (licensed)- This new service is the replacement for the old **utah** service and has the
> + most recent Google imagery for Utah.
> +
> + - **Basemap-Hybrid** (licensed)- This new service was created for convenience and performance for a dynamic mash-up of vector features with a transparent background from the **Basemap-Overlay** and the **Utah** imagery service.
> +
> +More information about the services from Discover can be found [here]({{"/data/sgid-base-map-services-arcmap/" | prepend:site.baseurl}}).
>
> here links are kind of rookie. Could you update that to something like
>
> More information about the services from Discover can be found in the Using
> SGID Base Map & Imagery Services in ArcMap <#m_559910426490274431_>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/agrc/gis.utah.gov/pull/384#pullrequestreview-23367487>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF6ebbzGBZNnZF-caehT6ey8sXz0k2Ogks5rfM0fgaJpZM4MJW2n>
> .
>

rkelson commented:
this PR is ready to merge
